services/publishing/base-repo/security/subject_extraction_rules.rego

```diff
@@ -0,0 +1,7 @@
+package rules
+
+default subject = { "user": null, "group": null }
+
+subject = { "user": input.upn, "group": "default"} {
```

Comment:
probably better to have input.sub as default

Reply:
Actually, sub doesn't have useful information in most cases.

Reply:
Isn't it the standard for subject?

Reply:
We'll do email if it's not empty and if it is we'll use sub.